traveler-ios-ui/TravelerKitUI/ViewControllers/BookablePassesViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  BookablePassesViewController.swift
+//  TravelerKit
+//
+//  Created by Ata Namvari on 2018-12-07.
+//  Copyright © 2018 Ata Namvari. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol BookablePassesViewControllerDelegate: class {
+    func bookablePassesViewControllerDidReceiveCheckout(withForm form: BookingForm?)
+}
+
+class BookablePassesViewController: UIViewController {
```

Comment:
Just `PassesViewController`

Reply:
The PassesViewController is the one that's embedded in it.